src/ray/raylet/worker.cc

```diff
@@ -153,7 +153,8 @@ void Worker::AssignTask(const Task &task, const ResourceIdSet &resource_id_set,
 
     auto message =
         protocol::CreateGetTaskReply(fbb, fbb.CreateString(spec.Serialize()),
-                                     fbb.CreateVector(resource_id_set_flatbuf));
+                                     protocol::CreateResourceIdSetInfos(
```

Comment:
Line 134 above should be changed from RAY_LOG(ERROR) to DEBUG, since the last actor task that kills the process will always count as a failure.